src/scripts/main.js

```diff
@@ -1,5 +1,46 @@
-'use strict';
+"use strict";
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.body.querySelector(".wall");
+const spider = document.body.querySelector(".spider");
+
+const halfSpiderX = spider.offsetWidth / 2;
+const halfSpiderY = spider.offsetHeight / 2;
+
+document.addEventListener("click", (e) => {
+  function moveAt(pageX, pageY) {
```

Comment:
You can declare this function in the upper scope and only call it inside eventListener